package protoFile.chat.chat1002;
import "protoFile/itemInfo.proto";

message chatMessageResponse{
     required int32 topic = 1;//消息的频道  1世界 2当前 3军团 4GM 5系统通告 6军团通告 7错误提示
     required int32 id = 2;//发送者的id
     required int32 profession = 3;//角色的职业
     required string fromName = 4;//角色的名称
     required string content = 5;//消息内容
     repeated ItemLinkData linkData = 6;//聊天连接信息
}
message ItemLinkData{
     optional int32 chatEquipType = 1;//超链接内容 0物品 1角色 2怪物
     optional int32 id = 2; //物品/角色/怪物的id
     optional string name = 3;//物品/角色/怪物的id
     optional ItemsInfo itemInfo = 4;//物品信息
}